Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/test/core/storage/TestCertificateContainer.kt

```diff
@@ -129,8 +129,11 @@ data class TestCertificateContainer(
             override val certificateCountry: String
                 get() = Locale(userLocale.language, testCertificate.certificateCountry.uppercase())
                     .getDisplayCountry(userLocale)
-            override val certificateId: String
-                get() = this@TestCertificateContainer.certificateId
+            override val qrcodeHash: String
+                get() = this@TestCertificateContainer.qrcodeHash
+
+            override val uniqueCertificateIdentifier: String
+                get() = testCertificateQRCode!!.hash
 
```

Comment:
this seems wrong, uniqueCertificateIdentifier should be the ci element

Reply:
Thanks! , Fixed